scene/gui/option_button.cpp

```diff
@@ -240,6 +240,21 @@ void OptionButton::set_item_tooltip(int p_idx, const String &p_tooltip) {
 	popup->set_item_tooltip(p_idx, p_tooltip);
 }
 
+void OptionButton::set_item_auto_translate_mode(int p_idx, AutoTranslateMode p_mode) {
+	if (p_idx < 0) {
+		p_idx += get_item_count();
+	}
+	if (popup->get_item_auto_translate_mode(p_idx) == p_mode) {
+		return;
+	}
```

Comment:
You can put this check before the above condition.

Reply:
The `get_item_xxx()` methods in these classes all have a quirk that they do not handle negative indices. So that condition should come first.